rllib/evaluation/episode_v2.py

```diff
@@ -254,16 +254,17 @@ def postprocess_episode(
         #  batches from the same policy to the postprocess methods.
         # Build SampleBatches for the given episode.
         pre_batches = {}
+
         for agent_id, collector in self._agent_collectors.items():
             # Build only if there is data and agent is part of given episode.
             if collector.agent_steps == 0:
                 continue
             pid = self.policy_for(agent_id)
             policy = self.policy_map[pid]
             pre_batch = collector.build_for_training(policy.view_requirements)
-            pre_batches[agent_id] = (pid, policy, pre_batch)
```

Comment:
why are we removing pid? Won't this cause any issue with `enable_connector=True`?

Reply:
ah you know what fixed in both places